internal/linker/link_node_modules.bzl

```diff
@@ -95,7 +96,18 @@ def get_module_mappings(label, attrs, vars, srcs = [], workspace_name = None):
                     fail(("duplicate module mapping at %s: %s maps to both %s and %s" %
                           (label, k, mappings[k], v)), "deps")
                 _debug(vars, "target %s propagating module mapping %s: %s" % (dep, k, v))
-                mappings[k] = v
+
+                # A package which was reachable transitively via a _binary
+                # target is assumed to be in the runfiles of that binary,
```

Comment:
nit: "via a _binary **rule**"

I was thrown off by `target` as I first assumed it meant target name